pyo3-bytes/src/bytes.rs

```diff
@@ -396,3 +401,39 @@ impl<'py> FromPyObject<'py> for PyBytesWrapper {
         Ok(Self(Some(buffer)))
     }
 }
+
+/// Implement Debug for PyBytes
+///
+/// Don't use the `bytes::Bytes` Debug impl, b/c it doesn't _look_
```

Comment:
I don't think it has to look exactly the same, as long as it's accurate. I'd rather reuse the upstream implementation. And if we use `br""` then the upstream debug string should fully round trip.

Reply:
I dont think it is possible to do the repr with `br""` feel free to try. 

- the current `Bytes` Debug impl outputs `b"..."` which precludes using `br`
- again, I dont think it is possible as it would lead to invalid evals
- the current impl DOES round trip and the tests handle all possible `u8` (AFAIK) 
- the python style with ascii chars appears to be a shorter string

Reply:
The problem is with '\' -- can't br with it as far as I know.